packages/commercetools/theme/middleware/commercetools.js

```diff
@@ -0,0 +1,16 @@
+import { createAccessToken } from '@vue-storefront/commercetools-api';
+import { setup } from '@vue-storefront/commercetools-api';
+import { config } from './../plugins/commercetools-config';
+
+export default async ({ app }) => {
+  let currentToken = app.$cookies.get('vsf-commercetools-token');
```

Comment:
what about to define `vsf-commercetools-token` in config or at least as a constant?